Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -0,0 +1,35 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "ABContactInternal.h"
+#import "UWP/WindowsApplicationModelContacts.h"
+
+@interface _ABContact ()
+
+@end
+
+@implementation _ABContact
+
+- (id)initWithContact:(WACContact*)contact {
+    self = [super init];
+    if (self) {
+        self.contact = contact;
```

Comment:
_ABContact is essentially a wrapper for contact, so an _ABContact wants strong ownership over its contact

---
In reply to: [73965363](https://github.com/Microsoft/WinObjC/pull/837#discussion_r73965363) [](ancestors = 73965363)